vignettes/customizing_styler.Rmd

```diff
@@ -50,17 +50,17 @@ As the name says, this function removes spaces after the opening parenthesis. Bu
 string_to_format <- "call( 3)"
 pd <- styler:::compute_parse_data_nested(string_to_format) %>%
   styler:::pre_visit_one(default_style_guide_attributes)
-pd$child[[1]] %>%
-  select(token, terminal, text, newlines, spaces)
+
+cols <- c('token', 'terminal', 'text', 'newlines', 'spaces')
+pd$child[[1]][, cols]
```

Comment:
maybe `subset(pd$child[[1]], select = cols)`?

Reply:
or since we've magrittr:

```r
pd$child[[1]] %>%
  subset(select = c(...))
```

Reply:
I don't know... the audience of the vignette is probably R developers who know `subset()` but I think it's one of these functions that are not used often anymore these days... `]` is not very pretty here I know but probably easier to understand for most.